### ISSUE TITLE
The updates of DDPG target network parameters in normalized layers

### ISSUE THREAD

**Issue by sasforce** [openai/baselines#647]
Hello, thanks for sharing. I wonder how to update target network parameters in normalized layers. Are they the same with neural network parameters?

**Comment by pzhokhov** [comment]
Hi @sasforce! Not sure I understand the question. We are using layer normalization, which does not have any extra parameters to sync between target and current networks. One could, in principle, use batch normalization instead; in which case I would assume the normalization parameters will also need to be updated via the soft update. Does that answer your question?

**Comment by sasforce** [comment]
Thank you for your reply!
I mean I am not really clear about the update of normalization layer parameters like gammas and betas in target networks, whatever BN or LN.

**Comment by pzhokhov** [comment]
I see. Yes, those are updated the same way the rest of the trainable parameters of the neural net - copy before the beginning of the training, and soft update during training (at least that's how it is implemented in baselines, and from the paper it seems to me that that was the intent)

**Comment by sasforce** [comment]
Thank you very much! I got it.
